DigitalExcellence/dex-frontend
# bugfix/437-close-project-modal-on-navigation

## Description

When a project-detail-modal was opened and the user went back using the browser navigation they would be navigated to the previous page. The expected behavior is to close the opened modal.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Open a project modal
2. Click the navigation back button (top-left of your browser or back button on your mouse)
3. The modal should close and you should still be on the overview page

## Link to issue

Closes: #437
